File: app/src/main/java/com/example/academyhomework/TemporaryActivity.java
package com.example.academyhomework;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class TemporaryActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_temporary);

        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.flForFirstFragment, new BlankFragment())
                .commit();

        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.flForSecFragment, new SecondBlankFragment())
                .commit();
    }
}

File: app/src/main/java/com/example/academyhomework/TemporaryActivity.kt
package com.example.academyhomework

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.academyhomework.databinding.ActivityTemporaryBinding

class TemporaryActivity : AppCompatActivity() {

    private lateinit var binding: ActivityTemporaryBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityTemporaryBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}